### ISSUE TITLE
Building chroot fails for main and 7.3

### ISSUE THREAD

**Issue by incaseoftrouble** [DOMjudge/domjudge#1202]
Long story short, I build the chroot based on `dj_make_chroot.in`. This "fails" with the following error:
```
Ign:1 https://security.debian.org bullseye/updates InRelease
Err:2 https://security.debian.org bullseye/updates Release
  404  Not Found [IP: 151.101.194.132 443]
Hit:3 https://deb.debian.org/debian bullseye InRelease
Get:4 https://deb.debian.org/debian bullseye/main Translation-en [6241 kB]
Reading package lists...
E: The repository 'https://security.debian.org bullseye/updates Release' does not have a Release file.
```

It should not be `bullseye/updates` but `bullseye-security` (changing this single line in `dj_make_chroot.in` fixes this issue (can submit a fix for this)

Two related notes:
 * The chroot installs pypy but python3 -- pypy is python2 and I guess it should be dropped in favor of pypy3, since python2 is EOL (can include that in the fix)
 * The build simply continues if an error arises, creating a half-empty chroot. Errors are easy to miss if they happen during the build process. Maybe one could somehow stop the build on errors? (at least this happens in the docker build, I don't know about other cases)

**Comment by eldering** [comment]
The `pypy3` issue was already fixed in 81466e2813e502f6f6958f3ff09742385605ff05.

The `dj_make_chroot` script already errors out on problems, but it seems that `debootstrap` doesn't throw a nonzero exitcode. We'd have to look into that.

**Comment by incaseoftrouble** [comment]
> The pypy3 issue was already fixed in 81466e2.

Oops, sorry, I was working on the 7.3 branch!

> but it seems that `debootstrap` doesn't throw a nonzero exitcode.

Yes, exactly. It also treats some things as `W`arning which really are errors. I had a `W: failed to execute mount -t proc proc /proc` (or similar, I don't remember the wording) somehwere during package installation, which stopped the whole custom script to be executed but debootstrap just happily continued installing.

(point being: Already on warnings, the script probably should fail or at least emphasize them)
